src/Json_Encode.res

```diff
@@ -14,9 +14,9 @@ external boolArray: array<bool> => Js.Json.t = "%identity"
 
 @val external null: Js.Json.t = "null"
 
-let array = (encode, arr) => arr->Js.Array2.map(encode)->jsonArray
+let array = encode => arr => arr->Js.Array2.map(x => encode(x))->jsonArray
```

Comment:
What's the reason for this change? Curried functions can no longer be passed around as first-class values?

Reply:
It's a dependency issue. `Js.Array2.map` is curried, so type inference gives a curried type. A type annotation would also work here.

Reply:
Hmm, I still don't understand what kind of dependency issue would affect inference here. no big deal though, I'm sure I'll find out once v11 drops!